### PR TITLE
Add tests for Edit, Clear all completed, completed status

### PR DESCRIPTION
**Pull request by Meri-MG**
In this pull request, we added several changes, created a test file for edit, completed status, clear all functions with mocked local storage.

### CODE REVIEW

File: src/interactive.test.js [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+        </li>
+        <button class="btn">Clear all Completed</button>
+        `;
+  test('function to check the complete statues false', () => {
+    const list = localStorageMock;
+    // addTodoToList('take a break', false, 0, list);
+    const task = document.querySelector('.items').innerText;
+    const element = document.querySelector('.checkbox');
+    const bool = false;
+    checkCompleted(list, task, element, bool);
```
Comment (rloterh) [inline src/interactive.test.js:55]: - [x] One test case is failing. Kindly ensure to improve upon this aspect.

**Failing test case:**
image

**Review by rloterh — CHANGES_REQUESTED** [review]
Hi Meri & Aku,

You did extremely great on this project! 👏 👏 
There are however a few issues that you still need to work on to go to the next project but you are almost there! 🚦  🚦  

## Good Points 👍  
- Good ReadMe
- No linter errors


## Required Changes ♻️
_Kindly check the comments under the review._

<br>
Cheers and Happy coding!👏 💻 🍷

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 2 more limited reviews per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/interactive.test.js (hunk 1 above)
Comment (aakbarkhan) [reply src/interactive.test.js:55]: Fixed the issue for the completed function. Thank you.

**Comment by helman101** [comment]
# STATUS: APPROVED 🟢 🥳 

Hi @Meri-MG  🥇 , 

### Congratulations!!! 🥳 🎉

You have successfully completed all the expectations for this project, there is nothing more to do here, now you can go on your way and keep improving your skills.

Cheers and Happy coding!👏👏👏

gif

If anything is not 100% clear please leave your questions or comments in the PR thread, we will be glad to help you 😄.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**